releases/branches.md

```diff
@@ -0,0 +1,20 @@
+# Branches
+
+This page describes branch conventions we follow.
```

Comment:
I had a longer chart with @TheoBrigitte about that and we failed to find a clear workflow to keep release branches without forcing us (or some automation) to cherry pick. We also couldn't think of any lost in terms of git history when we take @xh3b4sd approach to create temporary branches for creating releases based on not-latest-release.

With so many opinions here it's hard to me to move. I think the problem here is quite important and we need to find a consensus. That's why I'd like to reiterate on the topic. I need some time to prepare for the session and schedule one. I get that some people are already tired. If you are and you don't care enough feel free to skip the session.

Reply:
I still think we do not need to keep the stupid branches and that we only create more problems than we solve. I am happy to join and make a call. Let's get this shit done. 🚀

Reply:
Let's make a call and sort this out 👍

Reply:
Could someone try to explain exactly what the problem is in your view with release branches?

Reply:
@mig4: we have a call today at 12 BST. You are invited.

Reply:
No, I know, it's just during the last call I failed to understand what is the problem with them and wanted t have time to digest it a bit.